Mathlib/Algebra/Order/Floor.lean

```diff
@@ -1483,3 +1522,6 @@ def evalIntCeil : PositivityExt where eval {u α} _zα _pα e := do
   | _, _, _ => throwError "failed to match on Int.ceil application"
 
 end Mathlib.Meta.Positivity
+
+-- Pushed over the limit by deprecations
+set_option linter.style.longFile 1600
```

Comment:
Ouch, that's too bad. But this file looks like it deserves splitting anyway (look at those imports!) so I can live with this.

Reply:
(Were you planning on doing anything else with this file?)

Reply:
I have no further changes for this file planned in the immediate future.